CollapseLauncher/XAMLs/MainApp/Pages/SettingsPage.xaml.cs

```diff
@@ -577,7 +576,10 @@ private string Waifu2XToolTip
                 var tooltip = Lang._SettingsPage.Waifu2X_Help;
                 if (ImageLoaderHelper.Waifu2XStatus == Waifu2XStatus.CpuMode)
                     tooltip += "\n\n" + Lang._SettingsPage.Waifu2X_Warning_CpuMode;
-                if (ImageLoaderHelper.Waifu2XStatus == Waifu2XStatus.NotAvailable)
```

Comment:
This would be more readable as a switch case

Reply:
adjusted in https://github.com/CollapseLauncher/Collapse/pull/428/commits/71efeffd461ffae5a1bafa5be2bb23167dae36d6